docs/README-EDITING.md

```diff
@@ -0,0 +1,174 @@
+| `::: tip`                    | A note.                                                    |
+| `::: example [options]`      | Renders a code example as specified in [options].          |
+| `::: source-code-link [URL]` | Adds a source code link to the right of an API ref header. |
+
+### Adding code examples
+Using the `example` Markdown container, you can add code snippets that show the code's result:
+
+```js
+::: example #exampleId .class :preset --html 1 --js 2 --css 3  --no-edit --tab preview
+// `#example1` is the example's ID, for creating a Handsontable container
+    ```html
+    <div id="example1"></div>
+    ```
+    ```js
+    // code here
+    ```
+    ```css
+    /* custom css here */
+    ```
+:::
+```
+
+The `example` Markdown container offers the following options:
+
+| Option         | Required | Example         | Possible values                                            | Usage                                                                                              |
+|----------------|----------|-----------------|------------------------------------------------------------|----------------------------------------------------------------------------------------------------|
+| `#exampleId`   | Yes      | `#data-grid-1`  | String                                                     | Container's unique ID.                                                                             |
+| `.class`       | No       | `.new-class`    | String                                                     | Container's custom class.                                                                          |
+| `:preset`      | No       | `:react`        | [`preset`](.vuepress/handsontable-manager/dependencies.js) | Sets code dependencies.                                                                            |
+| `--js <pos>`   | No       | `--js 1`        | Positive integer<br>(default `1`)                          | Sets the JS code snippet's position<br>in the markdown container.                                     |
+| `--html <pos>` | No       | `--html 2`      | Positive integer<br>(default `0`)                          | Sets the HTML code snippet's position<br>in the markdown container.<br><br>`0` disables the HTML tab. |
+| `--css <pos>`  | No       | `--css 2`       | Positive integer<br>(default `0`)                          | Sets the CSS code snippet's position<br>in the markdown container.<br><br>`0` disables the CSS tab.   |
+| `--no-edit`    | No       | `--no-edit`     | `--no-edit`                                                | Removes the **Edit** button.                                                                       |
+| `--tab`        | No       | `--tab preview` | [`codeOption`](.vuepress/highlight.js) \| `preview`        | Sets a tab as open by default.                                                                     |
```

Comment:
This option doesn't work. I tried `preview`/`html`/`css`. Always sets `code` as the default open tab.

Reply:
It doesn't hold back the initial release. Let's do it at your earliest convenience @Siemienik